pages/dynamic-portal.tsx

```diff
@@ -552,6 +641,14 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
     orderBy: {
       createdAt: "desc",
     },
+    select: {
+      name: true,
+      type: true,
+      required: true,
+      dateFormat: true,
+      decimals: true,
+      enumOptions: true,
+    },
```

Comment:
Is selecting the fields to get around json parsing the timestamps?

Reply:
Yes, thinking this would be the better way instead of editing the createdAt when we dont even use it